scripts/alter-package-json.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+/**
+ modify mozilla/shield-studies-addon-template/package.json in place
+ as part of `shield init`
+*/
+
+var jsonPath  = process.argv[2];
+var dir = process.argv[3];
+
+console.log(jsonPath, dir);
+var p = require(jsonPath);
+
+var oldName = 'shield-studies-addon-template';
+var newName = ['shield-study-',''][~~dir.startsWith('shield-study-')]+dir;
```

Comment:
Also, this seems duplicated from [index.js:193](https://github.com/mozilla/shield-study-cli/pull/19/files#diff-168726dbe96b3ce427e7fedce31bb0bcR193) above. Not sure if we want to use some shared function somewhere versus duplicating the default naming logic.